crates/burn-core/src/nn/loss/poisson.rs

```diff
@@ -0,0 +1,389 @@
+use std::f32::consts::PI;
```

Comment:
For no-std we should use this instead

```suggestion
use core::f32::consts::PI;
```